
# HG changeset patch
# User doko@ubuntu.com
# Date 1270024644 -7200
# Node ID 94084cbf82d3960227b9252d40d286635d5cfefd
# Parent 6ff10812c44ae360727e752dd2173dc0133c77ea
Fix typo in IcedTeaNPPlugin.cc.

2010-03-31  Matthias Klose  <doko@ubuntu.com>

        * plugin/icedteanp/IcedTeaNPPlugin.cc (plugin_start_appletviewer):
        Fix typo.

--- a/ChangeLog	Wed Mar 31 02:58:12 2010 +0100
+++ b/ChangeLog	Wed Mar 31 10:37:24 2010 +0200
@@ -1,3 +1,8 @@ 2010-03-31  Andrew John Hughes  <ahughes
+2010-03-31  Matthias Klose  <doko@ubuntu.com>
+
+	* plugin/icedteanp/IcedTeaNPPlugin.cc (plugin_start_appletviewer):
+	Fix typo.
+
 2010-03-31  Andrew John Hughes  <ahughes@redhat.com>
 
 	* configure.ac: Bump version to 1.7.2.
--- a/plugin/icedteanp/IcedTeaNPPlugin.cc	Wed Mar 31 02:58:12 2010 +0100
+++ b/plugin/icedteanp/IcedTeaNPPlugin.cc	Wed Mar 31 10:37:24 2010 +0200
@@ -1463,7 +1463,7 @@ plugin_start_appletviewer (GCJPluginData
       command_line[5] = NULL;
    } else
    {
-       command_line = (gchar**) malloc(sizeof(gchar)*3);
+       command_line = (gchar**) malloc(sizeof(gchar*)*3);
        command_line[0] = g_strdup(appletviewer_executable);
        command_line[1] = g_strdup("sun.applet.PluginMain");
        command_line[2] = NULL;

